cypress/setUp/ngrok.js

```diff
@@ -0,0 +1,13 @@
+const ngrok = require('ngrok')   // eslint-disable-line
```

Comment:
why eslint is complaining on this line?

Reply:
Because it thought I use a `devDependency` (ngrok) in a non-test file. Probably I need to say to eslint somehow that this folder is testing, but I didn't do it yet.